solr/core/src/java/org/apache/solr/rest/RestManager.java

```diff
@@ -16,15 +16,33 @@
  */
 package org.apache.solr.rest;
 
+import org.apache.solr.common.SolrException;
```

Comment:
I'd prefer you configure your IDE to keep java.* up front.  FWIW this is in the IntelliJ config in the project.

Reply:
oops my bad, thanks for catching